codebuild/bin/grep_simple_mistakes.sh

```diff
@@ -147,6 +147,20 @@ for file in $S2N_FILES_ASSERT_VARIABLE_NAME_INDEX; do
   fi
 done
 
+#############################################
+# Assert that we didn't accidentally add an extra semicolon when ending a line.
+# This also errors when there is whitespace in-between semicolons, as that is an empty
+# statement and usually not purposeful.
+#############################################
+S2N_FILES_ASSERT_DOUBLE_SEMICOLON=$(find "$PWD" -type f -name "s2n*.[ch]" -not -path "*/bindings/*")
+for file in $S2N_FILES_ASSERT_DOUBLE_SEMICOLON; do
+  RESULT_DOUBLE_SEMICOLON=`grep -Ern ';[[:space:]]*;' $file`
+  if [ "${#RESULT_DOUBLE_SEMICOLON}" != "0" ]; then
+    FAILED=1
+    printf "\e[1;34mFound a double semicolon in $file:\e[0m\n$RESULT_DOUBLE_SEMICOLON\n\n"
```

Comment:
alternatively you could emit the number of doubles found, not a nit, just a style comment.